libs/langchain/langchain/vectorstores/milvus.py

```diff
@@ -822,7 +822,6 @@ def from_texts(
             index_params=index_params,
             search_params=search_params,
             drop_old=drop_old,
-            **kwargs,
         )
-        vector_db.add_texts(texts=texts, metadatas=metadatas)
+        vector_db.add_texts(texts=texts, metadatas=metadatas, **kwargs)
```

Comment:
there are other parameters for __init__ that we should now change to allow to pass in for backwards + full compatibility

```
primary_field: str = "pk",
        text_field: str = "text",
        vector_field: str = "vector",
```

Reply:
As I am seeing in previous versions, Args accepted by init:
```
        primary_field: str = "pk",
        text_field: str = "text",
        vector_field: str = "vector",
```
Should we add those back to backwards compatibility?